updated readme md

Akhil-ps-dev opened the pull request.

advaith-unnikrishnan reviewed (changes requested):
- Convert the links from text to Hyperlinks in **_Install and Setup, Resource pool, and Project pool section_**
-  Remove the lines **_40 and 47_**, it's part of the learning path template and not required in the final doc

Akhil-ps-dev commented:
Hello,
I have changed to hyperlinks and removed the 40th and 47th lines.

On Mon, Mar 27, 2023 at 1:24 PM Advaith U ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
>    - Convert the links from text to Hyperlinks in *Install and Setup,
>    Resource pool, and Project pool section*
>    - Remove the lines *40 and 47*, it's part of the learning path
>    template and not required in the final doc
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tinkerhub/maker-station/pull/10#pullrequestreview-1358534856>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUQ4J6PDI772NQ67GGW2GILW6FBTLANCNFSM6AAAAAAWGQ5F7Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
